### ISSUE TITLE
Turkey interconnectors not all on map

### ISSUE THREAD

**Issue by jzlcdh** [electricitymap/electricitymap-contrib#1311]
They don't all seem to be shown e.g. Bulgaria is important.

There is some info (in terms of kV) from 2016 in http://www.dunyaenerji.org.tr/wp-content/uploads/2018/03/EmreBengiSezginSenSunum.pdf saying 
Armenia and Georgia 220 kV
Bulgaria, Greece, Syria, Irak, Iran 154 kV
Azerbajan, Iran 380 kV

They are also listed under "ENTERKONNEKSİYONLAR" e.g. https://www.teias.gov.tr/tr/turkiye-bulgaristan

Trading with EU is by auction or PTR https://tcat.teias.gov.tr/

Day ahead market info at https://www.epias.com.tr/en/day-ahead-market/introduction

**Comment by alixunderplatz** [comment]
Woud be nice to find sources for all of them!

At the moment, Bulgaria, Greece and Georgia are the most relevant and have also been displayed in the past, as far as I remember.

Values for Greece and Georgia were consistent. Georgia has an HVDC interconnector with Turkey, so that one can also remain offline for longer periods.

Exchanges with Bulgaria tend to show a lot n/a values on [ENTSO-E](https://transparency.entsoe.eu/transmission-domain/physicalFlow/show?name=&defaultValue=false&viewType=TABLE&areaType=BORDER_CTA&atch=false&dateTime.dateTime=08.04.2018+00:00|CET|DAY&border.values=CTY|TR!CTA_CTA|10YTR-TEIAS----W_CTA_CTA|10YCA-BULGARIA-R&dateTime.timezone=CET_CEST&dateTime.timezone_input=CET+(UTC+1)+/+CEST+(UTC+2))

Maybe the data table of the Bulgarian TSO "ESO" is a good source for the exchange. I'll track that site to make sure, these are not scheduled exchanges but, real ones.

http://www.eso.bg/?did=124#%D0%9E%D0%BF%D0%B5%D1%80%D0%B0%D1%82%D0%B8%D0%B2%D0%BD%D0%B8%20%D0%B4%D0%B0%D0%BD%D0%BD%D0%B8

**Comment by jzlcdh** [comment]
Hi @alixunderplatz,

I don't understand why the Czech republic is in this news article [https://aa.com.tr/en/energy/electricity/turkeys-electricity-import-bill-down-by-60-in-2017/18686](url) as it is not a neighbouring country.

Yes the article shows you are right that Bulgaria is the most relevant - would it be a good idea/easy for me to add it?

Can you explain what you mean that the values for Greece were consistent? And should I raise a separate issue to request pricing be shown from the day ahead market mentioned above?

**Comment by alixunderplatz** [comment]
@jzlcdh 

* yeah, Czech Rep. is definitely a mistake in that article and makes no sense :)

* yes a sepearte issue for Turkey's [electricity price](https://www.epias.com.tr/en/intra-day-market/introduction) will be helpful.
Could you figure out which price is the market price, the "PTF" (market clearing price) or the "AOF" (weighted average price)? The difference is small, but which of them is finally used for the wholesale market?

![image](https://user-images.githubusercontent.com/25743609/38493978-69f202b4-3bf4-11e8-960d-98f317ebcf3b.png)

* By writing "Greece had consistent values", I mean the data which is reported to ENTSO-E (from where the exchange values are retrieved) was continuous and had a good availability in the past.
Electricity exchanges between Turkey and Greece are not shown at the moment for other reasons.
@corradio I guess, because data is [more than 2 hours delayed](https://transparency.entsoe.eu/transmission-domain/physicalFlow/show?name=&defaultValue=true&viewType=TABLE&areaType=BORDER_BZN&atch=false&dateTime.dateTime=09.04.2018+00:00|CET|DAY&border.values=CTY|10YGR-HTSO-----Y!BZN_BZN|10YGR-HTSO-----Y_BZN_BZN|10YTR-TEIAS----W&dateTime.timezone=CET_CEST&dateTime.timezone_input=CET+(UTC+1)+/+CEST+(UTC+2)) at the moment?

* Opposed to this, data for Turkey <-> Bulgaria interconnector has been:
  * unavailable in the younger past most of the time 
  * or was [only showing data for TR --> BG](https://transparency.entsoe.eu/transmission-domain/physicalFlow/show?name=&defaultValue=false&viewType=TABLE&areaType=BORDER_CTY&atch=false&dateTime.dateTime=16.03.2018+00:00|CET|DAY&border.values=CTY|10YCA-BULGARIA-R!CTY_CTY|10YCA-BULGARIA-R_CTY_CTY|TR&dateTime.timezone=CET_CEST&dateTime.timezone_input=CET+(UTC+1)+/+CEST+(UTC+2)), but BG --> TR was recently displayed as "n/e", which is a problem.

**Comment by jzlcdh** [comment]
It seems from items 49 to 52 in [the list of data items](https://seffaflik.epias.com.tr/transparency/resources/6363-16-EK-1.pdf) that hourly info on interconnections should be available to companies using the [API](https://www.epias.com.tr/wp-content/uploads/2018/02/EN_API_PORTAL_USER_MANUAL.pdf). Perhaps this issue could be assigned to someone who can register for the API by representing Electricity Map as a company?

**Comment by corradio** [comment]
@jpierz is there something I should do here? Send me instructions if I need to register to get a token to unblock this.

**Comment by jzlcdh** [comment]
There is a new document (small enough for Google Translate) at [https://www.teias.gov.tr/sites/default/files/2018-05/NetTransferKapasiteleri-17.05.2018.pdf]
with some details of the interconnectors. Hope that helps.

**Comment by jzlcdh** [comment]
Hope this helps: https://www.teias.gov.tr/en-US/interconnections

**Comment by jzlcdh** [comment]
For example clicking on Iraq shows 150 MW export capacity

**Comment by jzlcdh** [comment]
The official website does not show 2022 capacity but news reports say it is now 500 MW to Iraq https://www.rudaw.net/english/middleeast/iraq/120420223

**Comment by jzlcdh** [comment]
Scroll down Syria to see 100 MW export capacity in 2021 which is no doubt still operational
